docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,543 @@
+.. _schema_registry_tutorial:
+
+Confluent Schema Registry Tutorial
+==================================
+
+This tutorial provides an example on how to use |sr-long|.
+
+
+Overview
+~~~~~~~~
+
+This tutorial provides a step-by-step example to use |sr-long|.
+It walks through the steps to enable client applications to read and write Avro data with compatibility checks as schemas evolve.
+
+Benefits
+^^^^^^^^
+
+Kafka producers write data to Kafka topics and Kafka consumers read data from Kafka topics.
+There is an implicit "contract" that producers write data with a schema that can be read by consumers, even as producers and consumers evolve their schemas.
+|sr-long| helps ensure that this contract is met with compatibility checks.
+
+It is useful to think about schemas as APIs.
+Applications depend on APIs and expect any changes made to APIs are still compatible and applications can still run.
+Similarly, streaming applications depend on schemas and expect any changes made to schemas are still compatible and they can still run.
+Schema evolution requires compatibility checks to ensure that the producer-consumer contract is not broken. 
+This is where |sr-long| helps: it provides centralized schema management and compatibility checks as schemas evolve.
+
+For a more in-depth understanding of the benefits of Avro and |sr|, refer to the following resources:
+
+* `Why Avro For Kafka Data <https://www.confluent.io/blog/avro-kafka-data/>`_
+* `Yes, Virginia, You Really Do Need a Schema Registry <https://www.confluent.io/blog/schema-registry-kafka-stream-processing-yes-virginia-you-really-need-one/>`_
+
+Target Audience
+^^^^^^^^^^^^^^^
+
+The target audience is a developer writing Kafka streaming applications, who wants to build a robust application leveraging Avro data and |sr-long|.
+
+This tutorial is not meant to cover the operational aspects of running the |sr| service. For production deployments of |sr-long|, refer to :ref:`Schema Registry Operations<schemaregistry_operations>`.
+
+Before You Begin
+~~~~~~~~~~~~~~~~
+
+Prerequisites
+^^^^^^^^^^^^^
+
+Before proceeding with this tutorial
+
+#. Verify have installed on your local machine:
+
+   * Java 1.8 to run |cp|
+   * Maven to compile the client Java code
+   * `jq` tool to nicely format the results from querying the |sr| REST endpoint
+
+#. Use the :ref:`quickstart` to bring up |cp|. With a single-line command, you can have running on your local machine a basic Kafka cluster with |sr-long| and other services.
+
+   .. sourcecode:: bash
+
+      $ confluent start
```

Comment:
minor: Perhaps this should only go as far as `confluent start shchema-registry`? Since Connect, REST, KSQL, and C3 (1) aren't being used for this tutorial (2) host machine might be memory-strained to start _everything_. 

Thoughts?

Reply:
@cricket007 

The main to start the whole stack is for Control Center -- it has integration with Schema Registry.  For users who like UIs, Control Center is a great place to see the schema and inspect topics with deserialized Avro-data.  In the tutorial, there's even a screenshot from the UI.  

The second reason to keep it as is: it educate users on Confluent CLI and specifically that it can start all the services in the stack including Connect, REST, KSQL.  We've heard feedback from our users that starting individual Kafka services is hard, so this is a means to educate users that this CLI is available.

Your point about memory constraints is well-taken, but we find this less and less as user host machines become more and more performant over time.